sidekiq_metrics_test.go

```diff
@@ -0,0 +1,120 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+	"time"
+)
```

Comment:
Can you please order these using `goimports`? Thanks!

Reply:
@svanharmelen yes. done.